src/main/java/Parser.java

```diff
@@ -0,0 +1,42 @@
+public class Parser {
+    public static Command parse(String input) throws ToDoException, DeadlineException, EventException, 
+            CommandException {
+        if (input.equals("bye")) {
+            return new ExitCommand();
+        } else if (input.equals("list")) {
+            return new ListCommand();
+        } else if (input.startsWith("delete")) {
+            int indexOfTaskToDelete = Integer.parseInt(input.substring(7));
```

Comment:
There are some magic numbers here (e.g. 7). It is currently not apparent to a reviewer, why you chose these numbers. Would suggest you refactor, index parsing logic into their own static methods. 
1. That way, you could add comments to describe more about what the method does, and why chose those numbers.
2. Abstracting into individual methods would also make them unit testable, thus making your code less fragile

That said, the issues I have raised may not be urgent since they do not directly impact workability of your code. But they could be issues to keep in mind in the future 🐱 .